iron-list.html

```diff
@@ -209,6 +216,34 @@
       },
 
       /**
+       * When true, `iron-list` will render the items in a fixed-width grid.
+       * If true the `itemWidth` property will also need to be set
+       */
+      grid: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true
+      },
+
+      /**
+       * When `grid` is true this will define the width of each item in the
+       * grid. This plus the value of `itemOffset` will determine how many
+       * items are rendered per row
+       */
+      itemWidth: {

```

Comment:
What about if instead of an `itemWidth` property, we let the user set the width in CSS. e.g.

```html
<iron-list>
  <template>
    <div style="width: 20%">
      [[item.name]]
    </div>
  </template>
</iron-list>
```

`itemWidth` could become a private property such that:
```js
_updateMetrics: function(itemSet) {
...
this._itemWidth =  this._physicalCount > 0 ? this._physicalItems[0].offsetWidth : SOME_DEFAULT_VALUE; 
...
}
```

In that way:
1. Users can use media queries to set the item's width for different breakpoints.
2. Users can use percentages or pixels. 
3. The API would be consistent with how it obtains the item's height in the `!grid` case.